bin/ddev-share

```diff
@@ -21,7 +21,7 @@ echo "Waiting a few seconds for the service to come up"
 sleep 3
 NGROK_HOST=$(ngrok-url)
 wp-replace "${PROJECT_HOST}" "${NGROK_HOST}"
-
+echo "export const baseUrl = 'https://${NGROK_HOST}';" > tests/e2e/Shared/base-url.js
```

Comment:
This looks a little rough. Have you exlored alternative solutions before settling with this?

My impulse reaction would be to try and implement this via environment variables, but I'd have to try and see if this would actually be accessible to other scripts/shells.

I say this, because it "could" be as easy as an `export NGROK_HOST`. And in that case, it would be a spectacularly good idea to simply add this to [this PR](https://github.com/inpsyde/ddev-tools/pull/2/files) and then we could just remove this file entirely!

Reply:
Done in here, also with the parameter to conditionally wait or not. Wanted to show you the solution before attempting a PR in DDEV-tools. Not sure if the conditional part is something everybody would need...

Reply:
In said PR, I'd probably be more explicit and add multiple parameters like `--no-restore`, `--no-db` instead of a single-purpose catch-all